mmpose/__init__.py

```diff
@@ -3,8 +3,8 @@
 
 from .version import __version__, short_version
 
-mmcv_minimum_version = '1.1.3'
-mmcv_maximum_version = '1.3'
+mmcv_minimum_version = '1.2.6'
```

Comment:
Why is the minimum version changed

Reply:
I followed https://github.com/open-mmlab/mmaction2/pull/769/files

Reply:
So this should not be changed?

Reply:
Yeah keep the original one until someone reports incompaitibility